### ISSUE TITLE
Multiple page PDFs fail in 0.3.4

### ISSUE THREAD

**Issue by myrison** [plamola/ocr-joplin-notes#9]
Hi again @plamola.  Apologies for all of the recent traffic on this, but when I tested your fix yesterday, I just did it with a simple PDF & image test, both of which worked.

I've been seeing intermittent failures since yesterday and have now confirmed that the problem is with multi-page PDFs.  Below is an input & error code that hopefully helps you track down what is causing the errors in the new code.

I tested both locally and in the docker version and the same result occurs for both.  Thanks for taking a look when you have time.

```
iMac-5k:v0.3.4 Jason$ MAX_IMAGE_PIXELS=400000000 JOPLIN_TOKEN=<removed> python ./ocr_joplin_notes/cli.py --mode=FULL_RUN --tag=ocr-2-pagers
Environment variable JOPLIN_SERVER not set, using default value: http://localhost:41184
Mode: FULL_RUN
Tag: ocr-2-pagers
Exclude Tags: ()
Language: eng
Add previews: yes
Autorotation: yes
Starting OCR for tag ocr-2-pagers.
------------------------------------
note: 2-page PDFs
- file: 2-page-pdf-4.pdf [application/pdf]
Traceback (most recent call last):
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/./ocr_joplin_notes/cli.py", line 94, in <module>
    sys.exit(main())  # pragma: no cover
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/./ocr_joplin_notes/cli.py", line 84, in main
    res = run_ocr.run_mode(mode, tag, exclude_tags)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/run_ocr.py", line 78, in run_mode
    return full_run(tag, exclude_tags)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/run_ocr.py", line 90, in full_run
    return Joplin.perform_on_tagged_note_ids(perform_ocr_for_note, tag_id, exclude_tags)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/joplin_data_wrapper.py", line 84, in perform_on_tagged_note_ids
    usage_function(note_id)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/run_ocr.py", line 145, in perform_ocr_for_note
    result_tag = ocr_resources(note)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/run_ocr.py", line 159, in ocr_resources
    data = ocr_resource(resource, ADD_PREVIEWS and note.markup_language == 2)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/run_ocr.py", line 227, in ocr_resource
    ocr_result = file_ocr.extract_text_from_pdf(full_path, language=LANGUAGE)
  File "/Users/Jason/Documents/Seafile/Personal/Misc/docker/joplin-ocr/v0.3.4/ocr_joplin_notes/file_ocr.py", line 111, in extract_text_from_pdf
    embedded_text = "" + page.extractText()
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/PyPDF2/pdf.py", line 2593, in extractText
    content = self["/Contents"].getObject()
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/PyPDF2/generic.py", line 516, in __getitem__
    return dict.__getitem__(self, key).getObject()
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/PyPDF2/generic.py", line 178, in getObject
    return self.pdf.getObject(self).getObject()
  File "/Users/Jason/.pyenv/versions/3.9.2/lib/python3.9/site-packages/PyPDF2/pdf.py", line 1598, in getObject
    self.stream.seek(start, 0)
ValueError: seek of closed file
```
I can send the docker error log as well if it'd be helpful.

**Comment by plamola** [comment]
Should now be fixed in version 0.3.5

**Comment by myrison** [comment]
Brilliant, thank you again.
